Makefile

```diff
@@ -55,7 +55,7 @@ $(WEAVER_EXE) $(WEAVEDNS_EXE) $(WEAVEPROXY_EXE) $(NETCHECK_EXE): common/*.go com
 		false; \
 	}
 
-$(WEAVER_EXE): router/*.go ipam/*.go ipam/*/*.go prog/weaver/main.go
+$(WEAVER_EXE): router/*.go ipam/*.go ipam/*/*.go gossipdns/*.go prog/weaver/main.go
```

Comment:
Why is `gossipdns` and not a subdirectory in `router/`? It is part of the router now...